app/src/main/java/fr/free/nrw/commons/explore/ParentViewPager.java

```diff
@@ -0,0 +1,37 @@
+package fr.free.nrw.commons.explore;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import androidx.viewpager.widget.ViewPager;
+
+public class ParentViewPager extends ViewPager {
```

Comment:
Please add some Javadoc to explain what is the purpose of this class, for instance you can explain the bug briefly and say that how this class helps prevent it.
Also, Javadoc for each member and method, even if a super simple one, is our project rule. Thanks for your understanding :-)

Reply:
Javadoc added.